libs/sdk-ui-kit/src/utils/featureFlags.ts

```diff
@@ -0,0 +1,62 @@
+// (C) 2021 GoodData Corporation
+import { ISettings } from "@gooddata/sdk-backend-spi";
+
+const FREE = "free";
+const GROWTH = "growth";
+
+/**
+ * @internal
+ */
+
+export function shouldHidePPExperience(featureFlags: ISettings): boolean {
+    const hidePPExperience = Boolean(featureFlags.hidePixelPerfectExperience);
+    const enablePPExperience = featureFlags.enablePixelPerfectExperience;
+    const platformEdition = featureFlags.platformEdition;
+    const isFreemiumUser = isFreemiumEdition(platformEdition.toString());
+
+    return hidePPExperience || (isFreemiumUser && !enablePPExperience);
+}
+
+/**
+ * @internal
+ */
+
+export function isFreemiumEdition(platformEdition: string): boolean {
+    return platformEdition === FREE || platformEdition === GROWTH;
+}
+
+/**
+ * @internal
+ */
+
+export function generateSupportUrl(projectId = "", sessionId = "", userEmail = "", url = ""): string {
+    return [
+        "https://support.gooddata.com/hc/en-us/requests/new",
+        "?ticket_form_id=582387",
```

Comment:
What is this hard-coded ID? Is there a chance that it will change?

Reply:
https://github.com/gooddata/gdc-goodstrap/pull/1048/files#diff-22fc838a981890536566ac808f32f7261438a3495fc5767c1e845897bdf47fbfR552
Hi Dan I just saw in goodstrap we had this hardcode for some reason. Do you have any idea about this?

Reply:
Oh, I see it now, it seems to be some category ID for the support system. Thanks for the link, let's keep it as is.